tf/tfprocess.py

```diff
@@ -104,6 +117,13 @@ def __init__(self, cfg):
         self.RESIDUAL_BLOCKS = self.cfg['model']['residual_blocks']
         self.SE_ratio = self.cfg['model']['se_ratio']
         self.policy_channels = self.cfg['model'].get('policy_channels', 32)
+        self.policy_embedding_size = self.cfg['model'].get('policy_embedding_size', self.RESIDUAL_FILTERS)
+        self.encoder_layers = self.cfg['model'].get('encoder_layers', 1)
```

Comment:
rename to pol_ prefix both variable and the name in the cfg?
Same for encoder_d_model and encoder_dff